uiowa/uiowa
# Admissions Counselors (map, listing, person)

Resolves #2485
Resolves #1200 

![admissions local drupal uiowa edu_counselors](https://user-images.githubusercontent.com/4663676/99283613-dd1e7480-27fa-11eb-8c54-43b03626f6e4.png)

This is a custom map block for Admissions but takes the opportunity to add the latest Leaflet library to the uids_base build process for use in other situations.

When done and https://github.com/uiowa/uiowa/pull/2507 is merged in, it should provide Counselors info in the map. A click on the state updates the Counselors view block filter and displays a pop up info box. A link in the info box will take the user to  /counselors/{state_abbr}.

# How to test

- Take a look at admissions.dev.drupal.uiowa.edu/counselors (but don't touch)

OR

- `composer install && blt frontend && blt ds --site=admissions.uiowa.edu`
- Create a few person nodes with counselor person type and different combinations of territory values.
- As a webmaster in layout builder, you should see the Counselors Map block under More...
- Placed with the Counselors view block, this map will update the exposed filter on that view. A query string also works `/counselors?territory=IA`

**Note:** There are a few select options that are not on the map.
